src/net_processing.h

```diff
@@ -71,6 +74,8 @@ class PeerManager : public CValidationInterface, public NetEventsInterface
         //! Whether or not the internal RNG behaves deterministically (this is
         //! a test-only option).
         bool deterministic_rng{false};
+        //! Number of headers sent in one getheaders message result.
+        uint32_t max_headers_result{MAX_HEADERS_RESULTS};
```

Comment:
Better to split the peer manager changes out into a separate commit

Reply:
done